pkg/remotewrite/config.go

```diff
@@ -175,6 +186,17 @@ func ParseArg(arg string) (Config, error) {
 		c.KeepNameTag = null.BoolFrom(v)
 	}
 
+	if v, ok := params["headers"].(map[string]interface{}); ok {
+		for k, v := range v {
+			if v, ok := v.(string); ok {
+				if c.Headers == nil {
+					c.Headers = make(map[string]string)
+				}
```

Comment:
Because of the way this function is called this can be simplified down to:
```suggestion
        c.Headers = make(map[string]string)
	if v, ok := params["headers"].(map[string]interface{}); ok {
		for k, v := range v {
			if v, ok := v.(string); ok {			
```